__tests__/lint/no-invalid-media-type-examples/openapi.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: '3.0.0'
+info:
+  title: Example OpenAPI 3 definition.
+  version: 1.0
+  contact:
+    name: qa
+    url: https://swagger.io/specification/#definitions
+    email: email@redoc.ly
+servers:
+  - url: https://my_api.com
+
+paths:
+  /pet:
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                property:
+                  type: string
+            example:
+              $ref: '#/components/examples/State'
```

Comment:
Add one more tests that tests bad example to verify it reports location correctly. Thanks!